personal_assistant/pa_contacts/templates/contacts/index.html

```diff
@@ -13,55 +14,105 @@
+                            <div class="col" id="contact-{{ contact.id }}">
+                                <div class="card shadow-sm">
+                                    <div class="card-body">
+                                        <h5 class="card-title"> 
+                                            <i class="bi bi-person-fill"></i> {{ contact.name }} 
+                                        </h5>
+                                        <p class="card-text mb-1">
+                                            <i class="bi bi-telephone-fill"></i> {{ contact.phone }}
+                                        </p>
+                                        {% if contact.email %}
+                                        <p class="card-text mb-1">
+                                            <i class="bi bi-envelope-at-fill"></i> {{ contact.email }}
+                                        </p>
+                                        {% endif %}
+                                        {% if contact.address %}
+                                        <p class="card-text mb-1">
+                                            <i class="bi bi-house-fill"></i> {{ contact.address }}
+                                        </p>
+                                        {% endif %}
+                                        {% if contact.birthday %}
+                                        <p class="card-text mb-1">
+                                            <i class="bi bi-cake2-fill"></i> {{ contact.birthday }}
+                                        </p>
+                                        {% endif %}
+                                    </div>
+                                    <div class="card-footer">
+                                        <a 
+                                            href="{% url 'pa_contacts:delete' contact.id %}" 
+                                            role="button" 
+                                            class="btn btn-danger"
+                                        > 
+                                            <i class="bi bi-trash-fill"></i> Delete
+                                        </a>
+                                    </div>
+                                </div>
+                            </div>
+                        {% endfor %}
+                    </div>
+                </div>
+
+                <div class="col-md-3">
+                    <h4 class="text-center">Upcoming birthdays</h4>
+                    <div 
+                        class="btn-group mb-3 d-flex" 
+                        role="group" 
+                        aria-label="Birthday filters" 
+                        style="width: 100%;"
+                    >
+                        <a 
+                            href="?days=7" 
+                            class="btn btn-primary {% if days == 7 %}active{% endif %}"
+                        >
+                            7 days
```

Comment:
З меншою кількістю цифр виглядає наче приємніше хоча початковий варіант можна перенести в підказку (для більшої інформативності), а також можна спробувати підключити [додаткове оформлення](https://getbootstrap.com/docs/5.3/components/tooltips/):
```suggestion
                            title="Next 7 days"
                        >
                            Week
```